### ISSUE TITLE
Skungee on bungeecord error - can't read packet

### ISSUE THREAD

**Issue by Zulfen** [Skungee/Skungee#106]
Hey, I have been having trouble trying to hook up Skungee with 2 servers on the same node. 
This is the error I get in console:
```
 13:44:25 [INFO] [Skungee] [Skungee] Severe Error: [Could not read incoming packet]
> 13:44:25 [INFO] [Skungee] [Skungee] Severe Error: [Could not read incoming packet]
> 13:44:25 [INFO] [Skungee] [Skungee] Severe Error: [Could not read incoming packet]
> 13:44:25 [INFO]
> 13:44:25 [INFO]
> 13:44:25 [INFO]
> 13:44:25 [INFO] [Skungee] Something went wrong within Skungee.
> 13:44:25 [INFO] [Skungee] Something went wrong within Skungee.
> 13:44:25 [INFO] [Skungee] Something went wrong within Skungee.
> 13:44:25 [INFO] [Skungee] Please report this error to the developers of Skungee so we can fix this from happening in the future.
> 13:44:25 [INFO] [Skungee] Please report this error to the developers of Skungee so we can fix this from happening in the future.
> 13:44:25 [INFO] [Skungee] Please report this error to the developers of Skungee so we can fix this from happening in the future.
> 13:44:25 [INFO]
> 13:44:25 [INFO] [Skungee] Report this at https://github.com/TheLimeGlass/Skungee/issues. You can also message one of the Skungee developers.
> 13:44:25 [INFO]
> 13:44:25 [INFO] [Skungee] Stack trace:
> 13:44:25 [INFO]
> 13:44:25 [INFO] [Skungee] java.io.EOFException
> 13:44:25 [INFO] [Skungee] Report this at https://github.com/TheLimeGlass/Skungee/issues. You can also message one of the Skungee developers.
> 13:44:25 [INFO]
> 13:44:25 [INFO]
> 13:44:25 [INFO] [Skungee] Report this at https://github.com/TheLimeGlass/Skungee/issues. You can also message one of the Skungee developers.
> 13:44:25 [INFO]
> 13:44:25 [INFO] [Skungee] Stack trace:
> 13:44:25 [INFO] [Skungee] Stack trace:
> 13:44:25 [INFO] [Skungee] java.io.EOFException
> 13:44:25 [INFO] [Skungee] java.io.EOFException
> 13:44:25 [INFO] [Skungee] at java.io.ObjectInputStream$PeekInputStream.readFully(ObjectInputStream.java:2681)
> 13:44:25 [INFO] [Skungee] at java.io.ObjectInputStream$BlockDataInputStream.readShort(ObjectInputStream.java:3156)
> 13:44:25 [INFO] [Skungee] at java.io.ObjectInputStream.readStreamHeader(ObjectInputStream.java:862)
> 13:44:25 [INFO] [Skungee] at java.io.ObjectInputStream.<init>(ObjectInputStream.java:358)
> 13:44:25 [INFO] [Skungee] at me.limeglass.skungee.bungeecord.sockets.BungeeRunnable.run(BungeeRunnable.java:55)
> 13:44:25 [INFO] [Skungee] at java.io.ObjectInputStream$PeekInputStream.readFully(ObjectInputStream.java:2681)
> 13:44:25 [INFO] [Skungee] at java.lang.Thread.run(Thread.java:748)
> 13:44:25 [INFO] [Skungee] at java.io.ObjectInputStream$PeekInputStream.readFully(ObjectInputStream.java:2681)
> 13:44:25 [INFO]
> 13:44:25 [INFO] [Skungee] at java.io.ObjectInputStream$BlockDataInputStream.readShort(ObjectInputStream.java:3156)
> 13:44:25 [INFO] [Skungee] Information:
> 13:44:25 [INFO] [Skungee] at java.io.ObjectInputStream$BlockDataInputStream.readShort(ObjectInputStream.java:3156)
> 13:44:25 [INFO] [Skungee] at java.io.ObjectInputStream.readStreamHeader(ObjectInputStream.java:862)
> 13:44:25 [INFO] [Skungee] at java.io.ObjectInputStream.readStreamHeader(ObjectInputStream.java:862)
> 13:44:25 [INFO] [Skungee] at java.io.ObjectInputStream.<init>(ObjectInputStream.java:358)
> 13:44:25 [INFO] [Skungee] Skungee: 1.0.11
> 13:44:25 [INFO] [Skungee] at java.io.ObjectInputStream.<init>(ObjectInputStream.java:358)
> 13:44:25 [INFO] [Skungee] at me.limeglass.skungee.bungeecord.sockets.BungeeRunnable.run(BungeeRunnable.java:55)
> 13:44:25 [INFO] [Skungee] at me.limeglass.skungee.bungeecord.sockets.BungeeRunnable.run(BungeeRunnable.java:55)
> 13:44:25 [INFO] [Skungee] at java.lang.Thread.run(Thread.java:748)
> 13:44:25 [INFO] [Skungee] at java.lang.Thread.run(Thread.java:748)
> 13:44:25 [INFO]
> 13:44:25 [INFO]
> 13:44:25 [INFO] [Skungee] Information:
> 13:44:25 [INFO] [Skungee] Information:
> 13:44:25 [INFO] [Skungee] Bungee: git:BungeeCord-Bootstrap:1.14-SNAPSHOT:9e76966:1421
> 13:44:25 [INFO] [Skungee] Skungee: 1.0.11
> 13:44:25 [INFO] [Skungee] Skungee: 1.0.11
> 13:44:25 [INFO] [Skungee] Game version: 1.8.x-1.14.x
> 13:44:25 [INFO] [Skungee] Protocol version: 498
> 13:44:25 [INFO] [Skungee] Bungee: git:BungeeCord-Bootstrap:1.14-SNAPSHOT:9e76966:1421
> 13:44:25 [INFO] [Skungee] Bungee: git:BungeeCord-Bootstrap:1.14-SNAPSHOT:9e76966:1421
> 13:44:25 [INFO] [Skungee] Game version: 1.8.x-1.14.x
> 13:44:25 [INFO] [Skungee] Game version: 1.8.x-1.14.x
> 13:44:25 [INFO] [Skungee] Java: 1.8.0_212 (OpenJDK 64-Bit Server VM 25.212-b01)
> 13:44:25 [INFO] [Skungee] Protocol version: 498
> 13:44:25 [INFO] [Skungee] Protocol version: 498
> 13:44:25 [INFO] [Skungee] OS: Linux amd64 4.15.0-38-generic
> 13:44:25 [INFO]
> 13:44:25 [INFO] [Skungee] Java: 1.8.0_212 (OpenJDK 64-Bit Server VM 25.212-b01)
> 13:44:25 [INFO] [Skungee] Java: 1.8.0_212 (OpenJDK 64-Bit Server VM 25.212-b01)
> 13:44:25 [INFO] [Skungee] Thread: Thread[Thread-123,5,Skungee]
> 13:44:25 [INFO] [Skungee] OS: Linux amd64 4.15.0-38-generic
> 13:44:25 [INFO] [Skungee] OS: Linux amd64 4.15.0-38-generic
> 13:44:25 [INFO] [Skungee] Cause: [Could not read incoming packet]
> 13:44:25 [INFO]
> 13:44:25 [INFO]
> 13:44:25 [INFO]
> 13:44:25 [INFO] [Skungee] End of Error.
```
I have made sure everything in the config is correct. I believe it could be a firewall issue with my host, but they are not sure either.
I am using the latest version of Skungee.

**Comment by Zulfen** [comment]
Spigot Skungee:
https://pastebin.com/bhkZD3um
Bungeecord Skungee:
https://pastebin.com/VTPLVPbD
Bungeecord Config:
https://pastebin.com/Teb5uNfB

**Comment by Zulfen** [comment]
I am using Pterodactyl. I have heard it causes issues.

**Comment by ham1255** [comment]
Do you have abilty to setup more ports on the panel?

**Comment by Zulfen** [comment]
I have been given ports by my host. They are random and are on the the public IP.

**Comment by Zulfen** [comment]
![image](https://user-images.githubusercontent.com/38318382/69495542-fa5a8480-0ebf-11ea-8145-1631bb1e1122.png)

**Comment by Zulfen** [comment]
![image](https://user-images.githubusercontent.com/38318382/69495552-0e05eb00-0ec0-11ea-8e78-633be890cf96.png)

**Comment by ham1255** [comment]
Well aperantly it works for me here the setup
I used before i moved to java 

Bungeecord ports : server 25565,Skungee 1337 (i think)

Server1: server 1000,Skungee 7000
Server2: server 1001,Skungee 7001


Ports number depends on what ports you have 
And its recommneded to setup password as i saw the config.

**Comment by Zulfen** [comment]
Alright, will give one last try. Thanks for the quick response :)

**Comment by ham1255** [comment]
And in forgot disable auto port thing in skungee spigot config

**Comment by Zulfen** [comment]
By the way, I ran a copy of my servers locally and skungee works without errors. 🤔

**Comment by ham1255** [comment]
Because windows or any os doesnt block all ports like petrodacty do

**Comment by Mr-Vuda** [comment]
Have you get it to working?

**Comment by Zulfen** [comment]
Nah, not yet. I think its an issue with my host. Gonna talk to them.

**Comment by Mr-Vuda** [comment]
What host?

**Comment by Zulfen** [comment]
FreeMC

**Comment by Mr-Vuda** [comment]
You fixed it?

**Comment by Zulfen** [comment]
No, not yet. What firewall rules should I tell my host to add?

**Comment by TheLimeGlass** [comment]
Rather than using 0.0.0.0 for the binding address on Skungee Bungeecord, try using the direct address.

**Comment by Zulfen** [comment]
As in bind to my public IP? I have tried that and it just causes an error in console (can't bind to the address). When I have tried binding to the Pterodactyl docker it resolves back to 0.0.0.0 and binds to that.

**Comment by TheLimeGlass** [comment]
Duplicate https://github.com/Skungee/Skungee/issues/52
